src/test/regress/pg_regress_multi.pl

```diff
@@ -1019,21 +1021,19 @@ END
 
 my $exitcode = 0;
 
-sub PrepForVanillaTest
+sub RunVanillaTests
 {
     ###
     # We want to add is_citus_depended_object function to the default db.
-    # But without use-existing flag, pg_regress drops if exist and creates
-    # the default db. Thus, we set use-existing flag and manually create
-    # the default db, citus extension and the is_citus_depended_object
-    # function.
+    # But without use-existing flag, pg_regress recreates the default db
+    # after dropping it if exists. Thus, we set use-existing flag and
+    # manually create the default db, citus extension and is_citus_depended_object
+    # function. When use-existing flag is set, pg_regress does not drop
+    # default db.
     ###
 
-    my $dbName = shift;
-
-    # prepare tablespace folder
-    rmdir "./testtablespace";
```

Comment:
Hmm, on my local machine tablespace test is failing. I wonder if it is related to removing these?